packages/core-js-compat/src/build-entries.js

```diff
@@ -19,7 +18,7 @@ function getModulesForEntryPoint(entry) {
     const relative = resolve(dir, dependency);
     result.push(...getModulesForEntryPoint(relative));
   }
-  return [...intersection(new Set(result), order)];
+  return result.sort((a, b) => order.indexOf(a) > order.indexOf(b) ? 1 : -1);
```

Comment:
It's slow -/

Reply:
Oh I tought that you meant to remove `intersection` like this. Would it be ok to leave it there as it was before?

Reply:
Something like
```js
const resultSet = new Set(result);
return order.filter(it => resultSet.has(it));
```
could be better.

Reply:
`O(n)` vs `O(n^2 log n)` if I understood correctly.

Reply:
Yeah good idea.